### DISCUSSION TITLE
How to disable play button after first click?

### DISCUSSION THREAD

**Discussion by anggifergian** [lhz516/react-h5-audio-player#85] [General]
I wanted to use this package for my creat-react-app. However there is a requirement that user only can play the audio once. So how to make button disabled after user hit play button for the-first time?

**Comment by lhz516** [comment]
It's not a general feature. I would suggest you to use `customControlsSection` prop and overwrite `MAIN_CONTROLS` with your custom component. See: https://static.hanzluo.com/react-h5-audio-player-storybook/index.html?path=/docs/layouts-advanced--stacked
